_docs/tasks/security/per-service-mtls.md

```diff
@@ -9,7 +9,7 @@ type: markdown
 ---
 {% include home.html %}
 
-> If you are using Istio 0.7 or later, please refer to [authentication policy task]({{home}}/docs/tasks/security/authn-policy.html) for alternative (recommended) solution using authentication policy.
```

Comment:
Can we just remove the page for 0.8? People using 0.6 or earlier can go to archive.istio.io to find the version-appropriate docs.

Reply:
I am afraid this will cause confusion to people. They will wonder whether this is a mistake on website or it's not supported any more. What about keeping this page for a while and mentioning this will be deprecated, and then delete this page? I feel this will give people (who care about this feature) some time to switch.

Reply:
All righty....